### PR TITLE
Feature/split RP RS

### PR DESCRIPTION
**Pull request by oharsta**

### CODE REVIEW

File: roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2 [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        },
+        "^OrganizationDisplayName:({{ supported_language_codes | replace(',','|') }})$": {
+          "type": "string",
+          "info": "The friendly name of the organization. e.g. University of Harderwijk."
+        },
+        "^NameIDFormats:([0-2]{1})$": {
+          "$ref": "#/definitions/NameIDFormat",
+          "multiplicity": 3,
+          "info": "The NameIDFormat(s) this service provider supports. This can be transient, persistent or unspecified."
+        },
```
Comment (quartje) [inline roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2:126]: Do we need the NameIDFormats in addition to the NameIDFormat?

File: roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2 [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+          ],
+          "multiplicity": 4,
+          "sibblingIndependent": true,
+          "info": "From the pull down, select the type of contact."
+        }
+      },
+      "required": [
+        "name:en",
+        "secret",
+        "scopes"
```
Comment (quartje) [inline roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2:169]: Is this really required?

Comment (oharsta) [reply roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2:169]: Good question. We need 1 name and without secret the RS can not introspect. Scopes we can do without I guess.

File: roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2 (hunk 1 above)
Comment (oharsta) [reply roles/manage-server/files/metadata_configuration/oauth20_rs.schema.json.j2:126]: No, we don't. NameIDFormat is sufficient.

**Pull request merged**